src/code_80027D00.c

```diff
@@ -6597,66 +6594,42 @@ void func_800381AC(Player *player, struct Controller *controller, u8 arg2) {
 GLOBAL_ASM("asm/non_matchings/code_80027D00/func_800381AC.s")
 #endif
 
-#ifdef MIPS_TO_C
```

Comment:
Removing this ifdef might be failing the build. (The output rom doesn't match the baserom)

Try running `./diff func_800382DC` (add -mw to auto run when you make changes).
This will show you the difference in assembly between the original game and the decompiled. To get OK the assembly and registers need to match 1:1.

If you don't have `diff` you'll need to run `git submodule update --init --recursive` (as shown in the readme).


You can also run `python3 first-diff.py` to see what the first difference in rom is.